aws/resource_aws_api_gateway_rest_api.go

```diff
@@ -155,6 +171,19 @@ func resourceAwsApiGatewayRestApiUpdate(d *schema.ResourceData, meta interface{}
 	conn := meta.(*AWSClient).apigateway
 	log.Printf("[DEBUG] Updating API Gateway %s", d.Id())
 
+	if d.HasChange("body") {
+		if body, ok := d.GetOk("body"); ok {
+			_, err := conn.PutRestApi(&apigateway.PutRestApiInput{
+				RestApiId: aws.String(d.Id()),
+				Mode:      aws.String("overwrite"),
+				Body:      []byte(body.(string)),
+			})
+			if err != nil {
+				return err
```

Comment:
It does no harm, but I think the wrapper here isn't necessary as the only place the error will escalate to is the UI, where all that matters is string-based error message user will read, not the original error type.